Formula/podtnl.rb

```diff
@@ -0,0 +1,19 @@
+class Podtnl < Formula
+  desc "Expose your pod to Online from any kubernetes clusters"
+  homepage "https://podtnl.sh"
+  url "https://github.com/narendranathreddythota/podtnl/archive/1.0.tar.gz"
+  sha256 "e94d28fae97ee38ee3f4b3a502df5c510280eb49868aeb2cad40a7266158c0cc"
+  head "https://github.com/narendranathreddythota/podtnl.git"
+  depends_on "go" => :build
+  def install
+    ENV["GOPATH"] = buildpath
```

Comment:
Formula should probably use modules-based `go` build instead of `GOPATH`.

Reply:
Hello @alebcay Thank you I will correct it